runtime/src/lib.rs

```diff
@@ -683,11 +719,13 @@ macro_rules! construct_vln_runtime {
                     TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
                     Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
                     Aura: pallet_aura::{Config<T>, Pallet},
-                    Assets: pallet_assets::<Instance1>::{Pallet, Call, Storage, Event<T>},
-                    Fiat: pallet_assets::<Instance2>::{Pallet, Call, Storage, Event<T>},
+                    //Assets: pallet_assets::<Instance1>::{Pallet, Call, Storage, Event<T>},
+                    //Fiat: pallet_assets::<Instance2>::{Pallet, Call, Storage, Event<T>},
 
                     // vln dependencies
                     Whitelist: pallet_membership::{Call, Storage, Pallet, Event<T>, Config<T>},
+                    Tokens: orml_tokens::<Instance1>::{Config<T>, Event<T>, Pallet, Storage},
+                    Collateral: orml_tokens::<Instance2>::{Config<T>, Event<T>, Pallet, Storage},
```

Comment:
Can we still call the token collections `Assets` and `Fiat`? `Assets` being used for communities tokens and foreign assets like the `kUSD` and `Fiat` for the fungibles that can be traded via the escrow pallet? we no longer will "collateralize" the creation of other asset like previously with the `USDv`.